README.md

```diff
@@ -173,6 +172,123 @@ On Ubuntu/Debian:
+
+	mkdir mingw-w64-sofia-sip-git
+	cd mingw-w64-sofia-sip-git
+	wget https://raw.githubusercontent.com/Alexpux/MINGW-packages/master/mingw-w64-sofia-sip-git/PKGBUILD
+	wget https://raw.githubusercontent.com/Alexpux/MINGW-packages/master/mingw-w64-sofia-sip-git/0001-add-mingw-support.patch
+	wget https://raw.githubusercontent.com/Alexpux/MINGW-packages/master/mingw-w64-sofia-sip-git/0002-su-select-port.patch
+	wget https://raw.githubusercontent.com/Alexpux/MINGW-packages/master/mingw-w64-sofia-sip-git/0003-inet-pton-ntop-fix.patch
+	wget https://raw.githubusercontent.com/Alexpux/MINGW-packages/master/mingw-w64-sofia-sip-git/0004-add-su_win32_port.patch
+	wget https://raw.githubusercontent.com/Alexpux/MINGW-packages/master/mingw-w64-sofia-sip-git/0005-add-no-undefined-flag.patch
+	makepkg-mingw -sLfi --noconfirm
+
+libsrtp (x86, x86_64):
+
+	mkdir mingw-w64-libsrtp
+	cd mingw-w64-libsrtp
+	wget https://raw.githubusercontent.com/Alexpux/MINGW-packages/master/mingw-w64-libsrtp/PKGBUILD
+	wget https://raw.githubusercontent.com/Alexpux/MINGW-packages/master/mingw-w64-libsrtp/0001-DEV_URANDOM-removal.patch
+	wget https://raw.githubusercontent.com/Alexpux/MINGW-packages/master/mingw-w64-libsrtp/0002-symbolic-link-creation-removal-for-mingw-platform.patch
+	wget https://raw.githubusercontent.com/Alexpux/MINGW-packages/master/mingw-w64-libsrtp/0003-fix-path-for-getopt_s.patch
+	wget https://raw.githubusercontent.com/Alexpux/MINGW-packages/master/mingw-w64-libsrtp/words.tx
+	makepkg-mingw -sLfi --noconfirm
+
+usrsctp (x86, x86_64):
+
+	mkdir mingw-w64-usrsctp
+	cd mingw-w64-usrsctp
+	wget https://raw.githubusercontent.com/Alexpux/MINGW-packages/master/mingw-w64-usrsctp/PKGBUILD
+	wget https://raw.githubusercontent.com/Alexpux/MINGW-packages/master/mingw-w64-usrsctp/0001-add-mingw-support.patch
+	makepkg-mingw -sLfi --noconfirm
+
+rabbitmq-c (x86, x86_64):
+
+	mkdir mingw-w64-rabbitmq-c
+	cd mingw-w64-rabbitmq-c
+	wget https://raw.githubusercontent.com/Alexpux/MINGW-packages/master/mingw-w64-rabbitmq-c/PKGBUILD
+	wget https://raw.githubusercontent.com/Alexpux/MINGW-packages/master/mingw-w64-rabbitmq-c/0001-librabbitmq-CMakeLists.txt.patch
+	makepkg-mingw -sLfi --noconfirm
+
+* *Note:*
+
+If you need to build Janus Gateway documentation please install Windows version
+of Graphviz from
+
+	http://www.graphviz.org/Download_windows.php
+
+and make sure ```dot``` tool is in the PATH and
+
+	MSYS2_PATH_TYPE=inherit
+
+is not commented out in the mingw*.ini file.
+
+* *Note:*
+
+Windows version of Janus WebRTC Gateway fully supports Service Control
+Manager. You can create Janus WebRTC Gateway Windows Service by e.g.:
+
+	sc create "Janus WebRTC Gateway" binpath=c:/msys64/mingw64/bin/janus.exe
+
+To remove the service from Windows system type:
+
+	sc delete "Janus WebRTC Gateway"
+
+To start Janus WebRTC Windows Service execute:
+
+	sc start "Janus WebRTC Gateway"
+
+To stop Janus WebRTC Windows Service execute: 
+
+	sc stop "Janus WebRTC Gateway"
```

Comment:
Incredibly detailed documentation! Maybe too much... :smile: 
I'm wondering if the installation stuff should belong to a different .md file instead, so that the README is not too long. Anyway, just thinking out loud, and it is something we can worry about later.